packages/support/lib/npm.js

```diff
@@ -204,9 +204,10 @@ export class NPM {
      * "dummy" and is controlled by the user.  So we'll just add it as a dev
      * dep; whatever else it does is up to the user's npm config.
      */
-    const installOpts = (await hasAppiumDependency(cwd))
-      ? ['--save-dev']
-      : ['--save-dev', '--save-exact', '--global-style', '--no-package-lock'];
+    const installOpts = ['--save-dev', '--omit=peer'];
```

Comment:
`--omit=peer` should only be present if `hasAppiumDependency()` resolves `false`.  This will cause `npm` (unless a shrinkwrap says otherwise) to not install the `appium` peer dep.  If `hasAppiumDependency()` is `true`, we want to use `npm`'s default behavior as much as possible, since it's not _our_ `package.json` which is being modified.